lib/ui/painting/immutable_buffer.cc

```diff
@@ -30,6 +32,9 @@ ImmutableBuffer::~ImmutableBuffer() {}
 void ImmutableBuffer::RegisterNatives(tonic::DartLibraryNatives* natives) {
   natives->Register({{"ImmutableBuffer_init", ImmutableBuffer::init, 3, true},
                      FOR_EACH_BINDING(DART_REGISTER_NATIVE)});
+  natives->Register({{"ImmutableBuffer_fromAsset", ImmutableBuffer::fromAsset, 3, true},
```

Comment:
If we do it this way, we don't have to pass the `ImmutableBuffer` itself as an argument. We also don't want it to be static.